src/pages/ScriptPage.jsx

```diff
@@ -112,12 +119,39 @@ class Script extends React.Component {
     cytoscape.use(navigator);
     cytoscape.use(expandCollapse);
     cytoscape.warnings(false);
+    this._keepAlive = this._keepAlive.bind(this);
     this.renderCytoscapeElement = this.renderCytoscapeElement.bind(this);
     this.handleRenderChange = this.handleRenderChange.bind(this);
     this.handleValidateReport = this.handleValidateReport.bind(this);
     this.handleErrorModal = this.handleErrorModal.bind(this);
   }
 
+  componentWillMount() {
```

Comment:
I do not see that you would be closing websocket? I guess this can lead to memory leaks.

Maybe something like this should be included:
```
componentWillUnmount() {
  if (client.readyState === client.OPEN || client.readyState === client.CONNECTING) {
    client.close();
  }
}
```

Reply:
done